docs/image-style-props.md

```diff
@@ -81,7 +81,7 @@ const DisplayAnImageWithStyle = () => {
 const styles = StyleSheet.create({
   container: {
     display: 'flex',
-    flexDirection: 'vertical',
```

Comment:
This isn't even valid value is it?

Reply:
Nope lol

Reply:
Column is already the default thief which is probably why it wasn’t surfaced before.